libs/partners/huggingface/langchain_huggingface/chat_models/huggingface.py

```diff
@@ -113,6 +127,33 @@ def _convert_TGI_message_to_LC_message(
     return AIMessage(content=content, additional_kwargs=additional_kwargs)
 
 
+def _convert_hf_kwargs_llm_to_chat_completions(
+    **kwargs: Any,
+) -> Any:
+    """
+    Convert a set of keyword arguments to be compatible with [InferenceClient.chat_completion](https://huggingface.co/docs/huggingface_hub/v0.26.2/en/package_reference/inference_client#huggingface_hub.InferenceClient.chat_completion)
+    Overrides mapped keyword arguments and omits incompatible keyword arguments.
+    """
+    from huggingface_hub import InferenceClient  # type: ignore[import-untyped]
+
+    for k, v in LLM_INFERENCE_CLIENT_MAP.items():
+        if k in kwargs:
+            kwargs[v] = kwargs.pop(k)
+
+    sig = set(signature(InferenceClient.chat_completion).parameters)
+    valid_kwargs = {k: v for k, v in kwargs.items() if k in sig}
+    invalid_kwargs = {
+        k: v
+        for k, v in kwargs.items()
+        if k not in sig and k not in LLM_INFERENCE_CLIENT_EXCLUDED_KWARGS
```

Comment:
don't think `LLM_INFERENCE_CLIENT_EXCLUDED_KWARGS` is necessary if we're already checking the signature